src/assign.c

```diff
@@ -1151,7 +1151,7 @@ static R_len_t *savedtl=NULL, nalloc=0, nsaved=0;
 
 void savetl_init() {
   if (nsaved || nalloc || saveds || savedtl) {
-    error(_("Internal error: savetl_init checks failed (%d %d %p %p). please report to data.table issue tracker."), nsaved, nalloc, saveds, savedtl); // # nocov
+    INTERNAL_ERRORF("savetl_init checks failed (%d %d %p %p)", nsaved, nalloc, (void *)saveds, (void *)savedtl); // # nocov
```

Comment:
`saveds` is `SEXP *`, `savedtl` is `R_len_t *`, both caused warnings for `%p`, which IINM expects `void *`, hence the cast